scss/_patterns_rule.scss

```diff
@@ -0,0 +1,21 @@
+@import 'settings';
+
+$small-offset: map-get($grid-margin-widths, small);
+$default-offset: map-get($grid-margin-widths, default);
+
+@mixin vf-p-rule {
+  .p-rule {
+    @extend %hr;
+    @extend %u-no-margin--bottom--hr;
+    max-width: calc($grid-max-width - 2 * $small-offset);
+
+    @media (min-width: $threshold-4-6-col) {
+      max-width: calc($grid-max-width - 2 * $default-offset);
+    }
+
+    &--highlight {
```

Comment:
I think at some point we discussed not using such extrapolations of class names as a best practice, as it make it confusing and harder to find.

Could you move it below `.p-rule` simply as `.p-rule--highlight`?